projects/plutus/test/test_adc.c

```diff
@@ -0,0 +1,60 @@
+#include <inttypes.h>
+#include <stddef.h>
+#include <stdint.h>
+#include "interrupt.h"
+#include "log.h"
+#include "ltc_adc.h"
+#include "misc.h"
+#include "plutus_config.h"
+#include "soft_timer.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+#define TEST_ADC_NUM_SAMPLES 200
+
+LtcAdcSettings adc_settings = {
+  .mosi = { GPIO_PORT_B, 15 },  //
+  .miso = { GPIO_PORT_B, 14 },  //
+  .sclk = { GPIO_PORT_B, 13 },  //
+  .cs = { GPIO_PORT_B, 12 },    //
+
+  .spi_port = 1,                           //
```

Comment:
Can you just add some SPI ports to `spi_mcu.h`for x86 so we can stop doing this?